src/librustc_mir/transform/inline.rs

```diff
@@ -806,6 +806,7 @@ impl<'a, 'tcx> MutVisitor<'tcx> for Integrator<'a, 'tcx> {
                     *kind = TerminatorKind::Goto { target: tgt }
                 }
             }
+            TerminatorKind::Abort => { unimplemented!("Not sure what to do here?!"); }
```

Comment:
This should be a no-op - there are no targets to update.

Reply:
aka the same as `TerminatorKind::Unreachable`

Reply:
Thanks! Will fix for next version.